src/js/components/RadioButton/doc.js

```diff
@@ -68,6 +68,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '2px',
   },
+  'radioButton.check.backgroundColor': {
+    description: 'The background color of the checked icon in the RadioButton.',
+    type: 'string',
```

Comment:
```suggestion
    type: 'string | {dark: string, light: string}',
```
The color could also be a dark/light object.